src/kili/presentation/client/issue.py

```diff
@@ -0,0 +1,53 @@
+"""Client entrypoints methods for issues."""
+
+from itertools import repeat
+from typing import Dict, List, Literal, Optional
+
+import requests
+from typeguard import typechecked
+
+from kili.gateways.kili_api_gateway import KiliAPIGateway
+from kili.services.helpers import assert_all_arrays_have_same_size
+from kili.use_cases.issue import IssueUseCases
+from kili.use_cases.issue.types import IssueToCreateUseCaseInput
+from kili.utils.logcontext import for_all_methods, log_call
+
+
+@for_all_methods(log_call, exclude=["__init__"])
+class IssueClientMethods:
+    """Set of Issue mutations."""
```

Comment:
not only for mutations, all issue-related CRUD  operations right?

Reply:
Ah yeah good catch